README.md

```diff
@@ -118,8 +126,18 @@ Launch the wandbot with 8 workers. This speeds up evaluation
 WANDBOT_EVALUATION=1 gunicorn wandbot.api.app:app --bind 0.0.0.0:8000 --timeout=200 --workers=8 --worker-class uvicorn.workers.UvicornWorker
 ```
 
-Launch W&B Weave evaluation
 
+
+Set up for evaluation
+
+wandbot/src/wandbot/evaluation/config.py
+- evaluation_strategy_name : attribute name in Weave Evaluation dashboard
+- eval_dataset : 
+    - Latest English evaluation dataset: "weave:///wandbot/wandbot-eval/object/wandbot_eval_data:eCQQ0GjM077wi4ykTWYhLPRpuGIaXbMwUGEB7IyHlFU"
```

Comment:
It would be better to provide the link to the weave dataset webpage along with the actual weave url.